config/config.go

```diff
@@ -19,16 +20,15 @@ type Config struct {
 	OutgoingInstancesTopic         string        `envconfig:"DIMENSIONS_INSERTED_TOPIC"`
 	EventReporterTopic             string        `envconfig:"EVENT_REPORTER_TOPIC"`
 	DatasetAPIAddr                 string        `envconfig:"DATASET_API_ADDR"`
-	DatasetAPIAuthToken            string        `envconfig:"DATASET_API_AUTH_TOKEN"                 json:"-"`
-	ZebedeeURL                     string        `envconfig:"ZEBEDEE_URL"`
 	GracefulShutdownTimeout        time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval            time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	HealthCheckRecoveryInterval    time.Duration `envconfig:"HEALTHCHECK_RECOVERY_INTERVAL"`
```

Comment:
This should be `HealthCheckCriticalTimeout ` and `HEALTHCHECK_CRITICAL_TIMEOUT`

Reply:
fixed as part of task https://trello.com/c/rydxReH9/4736-update-default-and-environment-health-check-configurations-3d